Portal/src/Datahub.Infrastructure/Datahub.Infrastructure.csproj

```diff
@@ -9,11 +9,12 @@
     <ItemGroup>
       <PackageReference Include="Azure.Core" Version="1.49.*" />
       <PackageReference Include="Azure.Identity" Version="1.16.*" />
+      <PackageReference Include="Azure.ResourceManager" Version="1.13.2" />
```

Comment:
i was running into this one too. I have to learn how nuget packages work, but it looked like 1.13.* was defaulting to 1.13.0 and not .2

Reply:
puzzles me too... I wonder how much AI it will require to solve something like this lol